larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -403,7 +460,7 @@ inline void GetConvParamsType(const TfLiteBConv2DParams& conv_params,
   op_params.quantized_activation_max = conv_params.output_activation_max;
 }
 
-template <class T, class TBitpacked>
+template <typename T, typename TBitpacked, typename DstScalar>
 void EvalOpt(TfLiteContext* context, TfLiteNode* node,
              TfLiteBConv2DParams* params) {
```

Comment:
Right now it might be unclear what `T` refers to.
How about `<typename TIn, typename TBitpacked, typename TOut>` ?  Or `<typename SrcScalar, typename TBitpacked, typename DstScalar>` .

Reply:
Went with `SrcScalar`, in 21aac16.